src/Microsoft.TestPlatform.VsTestConsole.TranslationLayer/VsTestConsoleWrapper.cs

```diff
@@ -649,12 +663,26 @@ public async Task StartSessionAsync()
     public async Task<bool> StopTestSessionAsync(
         TestSessionInfo testSessionInfo,
         ITestSessionEventsHandler eventsHandler)
+    {
+        return await StopTestSessionAsync(
+            testSessionInfo,
+            options: null,
+            eventsHandler).ConfigureAwait(false);
+    }
+
+    /// <inheritdoc/>
+    [Obsolete("This API is not final yet and is subject to changes.", false)]
+    public async Task<bool> StopTestSessionAsync(
```

Comment:
Isn't it better to preserve the original signature order? (i.e. keep the event handler arg second and options last).